book.rb

```diff
@@ -0,0 +1,12 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Hi @thecodechaser, please let's add another method of adding rentals. To achieve the `has-many `relationship association, the book itself should be sent as a parameter to create the new rental. You may consider doing this:

```ruby
def add_rental(person, date)
      Rental.new(date, person, self)
end
```